src/auxiliary/math.jl

```diff
@@ -5,6 +5,101 @@
 @muladd begin
 #! format: noindent
 
+const TRIXI_UUID = UUID("a7f1ee26-1774-49b1-8366-f1abc58fbfcb")
+
+"""
+    set_sqrt_type(type; force = true)
+
+Set the `type` of the square root function to be used in `Trixi.jl`.
```

Comment:
```suggestion
Set the `type` of the square root function to be used in Trixi.jl.
```
We typically do not set Julia package names in code font.